cornac/models/spop/recom_spop.py

```diff
@@ -48,7 +48,7 @@ def fit(self, train_set, val_set=None):
         return self
 
     def score(self, user_idx, history_items, **kwargs):
-        item_scores = np.ones(self.total_items, dtype=np.float32)
+        item_scores = np.zeros(self.total_items, dtype=np.float32)
```

Comment:
What is the cause for this change? Shall we create a separate PR for this instead?

Reply:
I have created another PR for this fix.